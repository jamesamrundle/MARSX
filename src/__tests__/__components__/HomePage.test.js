import React from 'react';
import HomePage from '../../components/HomePage';
import renderer from 'react-test-renderer';

describe('<HomePage />', () => {
    it('should render correctly', () => {
        const tree = renderer.create(<HomePage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});