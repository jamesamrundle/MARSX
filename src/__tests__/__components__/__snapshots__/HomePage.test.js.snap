// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render correctly 1`] = `
<div
  className="home"
>
  <div
    className="header"
  >
    MARS
    <div
      className="subtitle"
    >
      Middleware for Assisting the Registration of Samples
    </div>
  </div>
</div>
`;
